src/features/contactverzoek/service.ts

```diff
@@ -85,6 +95,21 @@ export function saveContactverzoek({
     });
   }
 
+  function formatVraagAntwoordForToelichting(
+    vraagAntwoord: VraagAntwoord[]
+  ): string {
+    return vraagAntwoord.map((va) => `${va.vraag} - ${va.antwoord}`).join("\n");
```

Comment:
Heb je gecheckt of een geformatteerde string voldoende is? Lijkt me niet zo toekomstbestendig. Volgens mij kunnen we het objecttype prima flexibel maken zodat je er dynamische vraag/antwoord combinaties in kwijt kan.

Reply:
deze wordt bij 'interne toelichting' net voor de post in die textarea gezet daar en dan naar objecten gestuurd via de proxy. vandaar dat het hier een string is.

Reply:
geïmplementeerd volgens specs.